.github/ISSUE_TEMPLATE/releases/vs-tools-release.md

```diff
@@ -0,0 +1,30 @@
+# VS Tools Release
+
+VS Version: &lt;version&gt;
+
+## Tasks
+
+1. - [ ] Merge any pending PRs or commits from `dev` branch:
+      - [ ] &lt;placeholder link&gt;
+1. - [ ] Update the `vsVersion` in [vs-info.json](https://github.com/microsoft/dotnet-framework-docker/blob/master/eng/vs-info.json) to be the new VS version being released
+1. - [ ] Look up the latest [NuGet CLI versions](https://www.nuget.org/downloads) and update the [nuget-info.json](https://github.com/microsoft/dotnet-framework-docker/blob/master/eng/nuget-info.json) file with the latest patch of each major/minor version listed in the file
+1. - [ ] Wait for VS to be released
+1. - [ ] Install or update the new version of VS on your local machine
+1. - [ ] Create a zip file named `MSBuild.Microsoft.VisualStudio.Web.targets.<YYYY>.<MM>.zip` that contains the `MSBuild\Microsoft\VisualStudio\v16.0\Web` and `MSBuild\Microsoft\VisualStudio\v16.0\WebApplications` folders from the VS installation location
+1. - [ ] Upload the zip file to https://dotnetbinaries.blob.core.windows.net/dockerassets
+1. - [ ] Update the `webTargetsUrl` in [vs-info.json](https://github.com/microsoft/dotnet-framework-docker/blob/master/eng/vs-info.json) to reference the new zip file name
+1. - [ ] Gather the Build Tools and Test Agent download URLs by navigating to https://visualstudio.microsoft.com/downloads. Download links can be found under the `Tools for Visual Studio` section. Prior to clicking the download button, open a network profiler like your browser's dev tools or Fiddler so that you can capture the URL that gets downloaded. For `Build Tools for Visual Studio` look for a file named `vs_BuildTools.exe` being downloaded. For `Agents for Visual Studio`, ensure the radio button is defaulted to `Agent` and look for a file named `vs_TestAgent.exe`. Copy those two URLs and paste them into [vs-info.json](https://github.com/microsoft/dotnet-framework-docker/blob/master/eng/vs-info.json).
+1. - [ ] Run the `update-dependencies` tool to update all the necessary files:
+      - [ ] `dotnet run --project .\eng\update-dependencies\update-dependencies.csproj --datestamp-all <YYYYMMDD>`
+1. - [ ] Inspect generated changes for correctness
+1. - [ ] Commit generated changes
+1. - [ ] Create PR
+1. - [ ] Get PR signoff
+1. - [ ] Merge PR
+1. - [ ] Wait for changes to be mirrored to internal [dotnet-framework-docker repo](https://dev.azure.com/dnceng/internal/_git/dotnet-dotnet-framework-docker) (internal MSFT link)
+1. - [ ] Queue build of [dotnet-framework-docker pipeline](https://dev.azure.com/dnceng/internal/_build?definitionId=372) (internal MSFT link)
```

Comment:
This build should only be building the SDK images.  Additionally, the Samples do not need to be rebuilt because the runtime images haven't changed.

Reply:
We don't have support for only building the SDK images because of complications with the `REPO` arg getting set to the ACR location but needing to override that.  But we should be ensuring that only the SDK timestamps get changed.  I can make that adjustment.